examples/solvers/pdhg_denoising_L1_HuberTV.py

```diff
@@ -0,0 +1,123 @@
+"""Total variation denoising using PDHG.
+
+Solve the L1-HuberTV problem
+
+        min_{x >= 0} ||x - d||_1 + lam TV_gamma(x)
+
+where ``grad`` the spatial gradient and ``d`` is given noisy data.
+
+For further details and a description of the solution method used, see
+https://odlgroup.github.io/odl/guide/pdhg_guide.html in the ODL documentation.
+"""
+
+import numpy as np
+import scipy
+import odl
+import matplotlib.pyplot as plt
+
+# --- define setting --- #
+
+# Read test image: use only every second pixel, convert integer to float
+image = np.rot90(scipy.misc.ascent()[::2, ::2].astype('float'), 3)
+shape = image.shape
+
+# Rescale max to 1
+image /= image.max()
+
+# Discretized spaces
+space = odl.uniform_discr([0, 0], shape, shape)
+
+# Create space element of ground truth
+orig = space.element(image.copy())
+
+# Add noise and convert to space element
+noisy = odl.phantom.salt_pepper_noise(orig)
+
+# Gradient operator
+gradient = odl.Gradient(space, method='forward')
+
+# regularization parameter
+reg_param = 1
+
+# l1 data matching
+l1_norm = 1 / reg_param * odl.solvers.L1Norm(space).translated(noisy)
```

Comment:
why `1 / reg_param`

Reply:
The regularization parameter changes the strong convexity of f*. Thus, it is sometimes better to have it as part of g. Here it is 1, so it doesn't matter.